src/inmanta/deploy/tasks.py

```diff
@@ -89,6 +89,7 @@ async def get_executor(
             environment=task_manager.environment, model_version=version, agent_name=agent_name
         )
 
+        logger_for_agent(agent_name).debug("Retrieve code %s", code)  # TODO way too noisy: remove or trim
```

Comment:
TODO alert. Do you have a ticket for this?

Reply:
Removed.